openrtb_ext/imp_adnuntius.go

```diff
@@ -4,4 +4,5 @@ type ImpExtAdnunitus struct {
 	Auid      string `json:"auId"`
 	Network   string `json:"network"`
 	NoCookies bool   `json:noCookies`
+	MaxDeals  int    `json:maxDeals`
```

Comment:
couple of changes needed here:

- missing double quotes in json tag for `noCookies` and `maxDeals`
- should fix space to align `MaxDeals` with other members of `ImpExtAdnunitus`

```
type ImpExtAdnunitus struct {
	Auid      string `json:"auId"`
	Network   string `json:"network"`
	NoCookies bool   `json:"noCookies"`
	MaxDeals  int    `json:"maxDeals"`
}
```